Formula/apache-arrow-glib.rb

```diff
@@ -0,0 +1,39 @@
+class ApacheArrowGlib < Formula
+  desc "GObject Introspection files of Apache Arrow"
+  homepage "https://arrow.apache.org/"
+  url "https://www.apache.org/dyn/closer.cgi?path=arrow/arrow-0.7.1/apache-arrow-0.7.1.tar.gz"
+  sha256 "f8f114d427a8702791c18a26bdcc9df2a274b8388e08d2d8c73dd09dc08e888e"
+  head "https://github.com/apache/arrow.git"
+
+  depends_on "apache-arrow" => "0.7.1"
```

Comment:
We don't support depending on specific versions

Reply:
So, is there no way to force the version of depending apache-arrow?

Reply:
Nope. It has to use whatever the current version in the other formula is.

Reply:
I fixed it.